content/projects/gsoc/2020/project-ideas/jenkins-distribution-customize-service.adoc

```diff
@@ -20,7 +20,7 @@ For many users, they download Jenkins first, then select some plugins and config
 It might take a lot of time, like hours. But if we can get a perfect Jenkins distribution which contains all we need, 
 it can save that time for us.
 
-The service could be in the form of a website offered as https://customize.jenkins.io.
+The service could be in the form of a website offered as \https://customize.jenkins.io.
```

Comment:
```suggestion
The service could be in the form of a website offered as `https://customize.jenkins.io`
```

maybe?

Reply:
This does not work :(

Reply:
ughm 🤦

Reply:
your screen shot is missing the full-stop, is that because of this change? it makes the sentence not finish and has Plus capitalised

Reply:
I think I took the screenshot and left the full stop. I guess I will change it to not have the full stop